torchao/testing/model_architectures.py

```diff
@@ -177,3 +178,63 @@ def create_model_and_input_data(
     else:
         raise ValueError(f"Unknown model type: {model_type}")
     return model, input_data
+
+
+class Llama4Experts(nn.Module):
```

Comment:
would be good for this to actual match Llama4

https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama4/modeling_llama4.py#L45-L74

since we've adopted those attribute and parameter names for the AOQuantizable modules